### PR TITLE
[LTTP] Remove some more difficult settings

### PR DESCRIPTION
**Pull request by t3hf1gm3nt**
Removes weights from insanity ER, moving them to full, and expert item pool/functionality and moves them to hard. Was discussed that these settings are a bit much for a filler yaml and should be left for people to select for customs if they wish.

### CODE REVIEW

**Comment by ScootyPuffJr1** [comment]
Changes seem fine to me.

**Comment by alwaysintreble** [comment]
killable thieves should probably be a trigger and ganon_ped should be removed since we have local_ganon_ped. i'd also like to see some shop shuffle options but that might just be me

**Comment by t3hf1gm3nt** [comment]
I'll admit all these things are outside the scope of my original intention for this update but I'll answer my thoughts anyway.

Killable thieves as a trigger to only be on when enemizer is on? That's something I could agree too.

What do you mean by local Ganon ped? Ganon ped is always local since dungeon prizes are always local

I never play with shop shuffle so I'd have no idea what good options would be for those. I'll leave that to someone with better experience with it to submit a PR, but I could be convinced to have some options in there for it

**Comment by alwaysintreble** [comment]
i got confused with tfh 🤦

**Comment by TheT113** [comment]
I like all of these changes. 👍

File: games/A Link to the Past.yaml [hunk 1]
```diff
@@ -176,3 +176,11 @@ A Link to the Past:
   glitch_boots:
     on: 50 # Start with Pegasus Boots in any glitched logic mode that makes use of them
     off: 0
+triggers:
```
Comment (ThePhar) [inline games/A Link to the Past.yaml:179]: This needs to be at the same level as all the other options or it will fail to generate a yaml.

**Review by ThePhar — CHANGES_REQUESTED** [review]

File: games/A Link to the Past.yaml (hunk 1 above)
Comment (t3hf1gm3nt) [reply games/A Link to the Past.yaml]: indented over. look good now?

**Comment by ScootyPuffJr1** [comment]
Triggers look good

**Pull request merged**